src/command.c

```diff
@@ -83,33 +83,35 @@ static bool cmd_shutdown_bmda(target_s *t, int argc, const char **argv);
 const command_s cmd_list[] = {
 	{"version", cmd_version, "Display firmware version info"},
 	{"help", cmd_help, "Display help for monitor commands"},
-	{"jtag_scan", cmd_jtag_scan, "Scan JTAG chain for devices"},
-	{"swdp_scan", cmd_swdp_scan, "Scan SW-DP for devices"},
+	{"jtag_scan", cmd_jtag_scan, "Scan JTAG chain for devices: [IR_LENGTH ...]"},
```

Comment:
While this is correct, we're also dropping the parameter in #1477 so you can drop this particular change to help save some space